libraries/AP_Mount/AP_Mount_Backend.cpp

```diff
@@ -150,6 +150,10 @@ void AP_Mount_Backend::calc_angle_to_location(const struct Location &target, Vec
     // pan calcs
     if (calc_pan) {
         // calc absolute heading and then onvert to vehicle relative yaw
-        angles_to_target_rad.z = wrap_PI(atan2f(GPS_vector_x, GPS_vector_y) - _frontend._ahrs.yaw);
+        angles_to_target_rad.z = atan2f(GPS_vector_x, GPS_vector_y);
```

Comment:
I think not: atan2f return value is in the range of [-pi, pi] so it should do without any additional checks.